catroid/src/org/catrobat/catroid/formulaeditor/SensorHandler.java

```diff
@@ -337,6 +356,20 @@ public static Double getSensorValue(Sensors sensor) {
 				}
 			case LOUDNESS:
 				return Double.valueOf(instance.loudness);
+			case DATE_YEAR:
+				return Double.valueOf(instance.dateYear);
+			case DATE_MONTH:
+				return Double.valueOf(instance.dateMonth);
+			case DATE_DAY:
+				return Double.valueOf(instance.dateDay);
+			case DATE_WEEKDAY:
+				return Double.valueOf(instance.dateWeekday);
```

Comment:
here you could use 
```java
Double.valueOf(Calendar.getInstance().get(Calendar.DAY_OF_WEEK ));
```
returns 1 for sunday, 2 for monday, ...
you might want to  adapt this if you want to start the week with a monday (monday = 1, sunday = 7)